### ISSUE TITLE
Unmanaged dlls wont be loaded if their name contains dots.

### ISSUE THREAD

**Issue by redknightlois** [Fody/Costura#100]
I wanted to embed a dll named: "CudaLearn.Native.Viennacl.dll" and couldn't make it work. Changed it to "vcltensormath.dll" and suddenly everything started working.

**Comment by distantcam** [comment]
Was vcltensormath.dll still in your build folder?

**Comment by redknightlois** [comment]
I am building the .dll as part of an unmanaged wrapper so I just needed to change the output name to not include dots to make it work.
 
![including-dll](https://cloud.githubusercontent.com/assets/1045411/5601632/eb158ace-92eb-11e4-9af7-a928eb9cd09d.png)

The rest is exactly the same configuration. 

![solution-dll](https://cloud.githubusercontent.com/assets/1045411/5601630/ce6bc05a-92eb-11e4-8148-f90942ad5b04.png)

The configuration works as is. I started to distrust the loader because I had another unmanaged embedded in another project at the same solution. That's why I decided to go with the most simple version "a.dll" where it worked. After that settled for vcltensormath.dll as a workaround instead of the project name (the one that includes dots) and then open the issue.
